.devcontainer/setup.sh

```
@@ -0,0 +1,147 @@
+
+# Run migrations
+print_status "Running database migrations..."
+if mix ecto.migrate; then
+    print_success "Database migrations completed successfully"
+else
+    print_warning "Some migrations may have failed - this might be expected in development"
+fi
+
+# Check if pg_cron is working
+print_status "Verifying pg_cron functionality..."
+if psql $DATABASE_URL -c "SELECT check_pg_cron_status();" >/dev/null 2>&1; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)